### PR TITLE
Fix some of the text explaining mozharness

### PR DESCRIPTION
**Pull request by armenzg**
@lundjordan when you have a chance (no rush), would you mind having a look at this?

### CODE REVIEW

File: software.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Buildbot Overview
 
 If you are new to Buildbot, please follow this excellent tutorial:  `Buildbot in 5 min`_. If you end
 up lost or want more information on Buildbot itself, check out the full `Buildbot Docs`_.
-
+ss
```
Comment (djmitche) [inline software.rst]: ??

Comment (armenzg) [inline software.rst]: Unintentional!

**Comment by Callek** [comment]
@armenzg if you fix that unintentional thing are we going to merge this?

**Comment by armenzg** [comment]
I think I fixed the issue.
@Callek How do we review things around here?

**Comment by Callek** [comment]
@armenzg frankly, I'm unsure -- I never touched this repo before now. @lundjordan any insight or shall we just press the magic "Merge pull request" button and move on?

**Comment by Callek** [comment]
No response, I'm merging

**Pull request merged**